config/routes.rb

```diff
@@ -83,9 +83,7 @@ def matches?(_)
   get '(:lang)/listings/for-rent' => 'directory#rent', lang: /(en|es|zh|tl)/
   get '(:lang)/listings/for-sale' => 'directory#sale', lang: /(en|es|zh|tl)/
   # TODO: Paths on Bloom needs to be configurable
-  get '(:lang)/listing/:id', lang: /(en|es|zh|tl)/, to: redirect { |params, _|
-    "#{params[:lang] ? params[:lang] + '/' : ''}/listings/#{params[:id]}"
-  }
+  get '(:lang)/listings/:id' => 'listing#index', lang: /(en|es|zh|tl)/
```

Comment:
curious about what this change is.

Reply:
The page doesn't load / work with the previous redirect, but I'm honestly not sure what the exact issue was even fixing the `listings` --> `listing` typo

Reply:
ok interesting